bin/oneoff/remove_non_teacher_application_ids.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+
+puts "Removing application ids that are not teacher applications ..."
+modified_enrollments = []
+
+Pd::Enrollment.where.not(application_id: nil).each do |enrollment|
+  original_app_id = enrollment.application_id
+
+  begin
+    application = Pd::Application::ApplicationBase.find(original_app_id)
+    if application.type != "Pd::Application::TeacherApplication"
+      # The method set_application_id is triggered before a save
+      # Save the enrollment to re-compute set_application_id
+      enrollment.save!
+      modified_enrollments << enrollment.id
```

Comment:
this if block will only get executed for principal applications, right? why are we updating principal applications if the problem is with facilitator applications?

Reply:
ok, I think I missed some background here (meaning, I think you shared about this previously but now I am forgetting) -- if this can wait til tomorrow's checkin I'd love to talk about this in realtime to understand what's going on here.